manifests/overlays/superset-role/cluster-role-patch.yaml

```diff
@@ -59,25 +42,14 @@ rules:
   resources:
   - services/status
   verbs:
-  - list
   - patch
   - update
-  - watch
```

Comment:
I don't think list is possible with subresources, and I'm not sure about watch but we definitely don't do it.